game-core/src/test/java/org/triplea/lobby/common/DbUserTest.java

```diff
@@ -91,6 +91,7 @@ public void userNameValidationWithInvalidNames() {
         "",
         "a",
         "ab", // still too short
+        "12345678901234567890123456789012345678901", // one character too long
```

Comment:
Any thoughts to generating this string to avoid manually counting of the lenght?

EG:
```
    Strings.padEnd("", 41, '1');
```

Reply:
e98fda97b